haiku-apps/wakeup/wakeup-1.0.recipe

```diff
@@ -0,0 +1,39 @@
+SUMMARY="An application that plays a sound after given time"
+DESCRIPTION="WakeUp is a timer application that allows users to set their own ringtone,\
```

Comment:
Make sure no line is longer than 80 characters. When you escape a linefeed with "\", you mustn't forget to a space, otherwise words would be written together, or in this case the space after the "," would be missing.

When you try out your recipe by building it with haikuporter, open the resulting HPKG in HaikuDepot. You'll better see where spaces are missing or in some cases, where a linebreak or even empty line may look nicer.